aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -79,7 +79,7 @@ define_gas_parameters_for_transaction!(
     [
         max_transaction_size_in_bytes,
         "max_transaction_size_in_bytes",
-        8192
+        8192 * 16,
```

Comment:
Needed this to get `smoke-test::aptos::gas-check` to pass:

```
cargo nextest run --profile ci --package smoke-test --test-threads 6 --retries 3 -- smoke-test::aptos::gas-check
```

It failed with:
```
Error: Request failed: RestError { code: 400, message: "invalid transaction: EXCEEDED_MAX_TRANSACTION_SIZE", aptos_ledger_version: None }
```

I think due to the large size of the gas parameters involved in BLS signatures.